Social/Facebook/Oauth.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/*
+ * This file is part of the Integrated package.
+ *
+ * (c) e-Active B.V. <integrated@e-active.nl>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Integrated\Bundle\SocialBundle\Social\Facebook;
+
+use Facebook\Exceptions\FacebookResponseException;
+use Facebook\Exceptions\FacebookSDKException;
+use Facebook\Facebook;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class Oauth
+ * @package Integrated\Bundle\SocialBundle\Social\Facebook
+ */
+class Oauth
```

Comment:
Can you add an Interface for this class ( also for the Twitter ) they both use callback / login